src/js/components/Notification/__tests__/Notification-test.tsx

```diff
@@ -63,30 +64,28 @@ describe('Notification', () => {
     expect(onClose).toBeCalled();
   });
 
-  [
+  const validPositions: LayerPositionType[] = [
     'top',
     'bottom',
     'left',
     'right',
-    'start',
-    'end',
```

Comment:
Removed 'start' and 'end' as they are not valid string literals within the defined type  LayerPositionType.

Reply:
I think the LayerPositionType should be change to include 'start' and 'end'

Reply:
yes I will include it.